common/djangoapps/student/views.py

```diff
@@ -2220,14 +2225,16 @@ def password_reset_confirm_wrapper(request, uidb36=None, token=None):
         )
 
     if request.method == 'POST':
-        password = request.POST['new_password1']
-        is_password_valid, password_err_msg = validate_password(user, password)
+        password1 = request.POST['new_password1']
+        password2 = request.POST['new_password2']
```

Comment:
I believe django's implementation of `password_reset_confirm` already verifies that both new passwords match.  So it seems redundant to do this here.